.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Force to use trusty for now, as ubuntu 16.04 became default recently. Test case "can listen for the creation of a deeply nested file" randomly fail and I couldn't reproduce in my local machine.

source: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

Reply:
Can you remove this please? We should be testing on xenial, and it occasionally fails on trusty as well :(.